atomic_reactor/utils/koji.py

```diff
@@ -340,12 +341,20 @@ def get_image_output(workflow, image_id, arch):
     :return: tuple, (metadata dict, Output instance)
 
     """
-    saved_image = workflow.exported_image_sequence[-1].get('path')
     image_name = get_image_upload_filename(workflow.exported_image_sequence[-1],
                                            image_id, arch)
 
-    metadata = get_output_metadata(saved_image, image_name)
-    output = Output(file=open(saved_image), metadata=metadata)
+    # Provide empty tar file for docker archive metadata
```

Comment:
Is it OK to provide empty archive directly in here. Or should I create a separate method in some helper python file e.g. utils

Reply:
If we're going to add the README then I would prefer a separate method, otherwise I don't mind as much

Reply:
I created separate method for it.